students/yooyoung/users/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.core.validators import RegexValidator
+
+
+class User(models.Model):
+    name = models.CharField(max_length=20)
+    email = models.EmailField(max_length=50, unique=True)
```

Comment:
```suggestion
    email = models.EmailField(max_length=200, unique=True)
```
max_length는 넉넉하게 지정해주시는 게 좋습니다.